src/sentry/api/helpers/group_index.py

```diff
@@ -343,6 +343,7 @@ def _delete_groups(request, project, group_list, delete_type):
     transaction_id = uuid4().hex
 
     GroupHash.objects.filter(project_id=project.id, group__id__in=group_ids).delete()
+    GroupInbox.objects.filter(project_id=project.id, group__id__in=group_ids).delete()
```

Comment:
Probably don't really need to filter on `project_id` here since we have an index on `group_id`, but just matching patterns